src/simplify_qdq.cpp

```diff
@@ -55,12 +55,12 @@ std::unordered_set<std::string> get_quantizable_op_names()
 struct match_find_quantizable_ops
 {
     static bool
-    is_valid_scale(instruction_ref scale, std::vector<std::size_t> lens, std::size_t axis)
+    is_valid_qparam(instruction_ref scale, std::vector<std::size_t> lens, std::size_t axis)
```

Comment:
nit: this is used for both scale and zp. having scale as the parameter name is not a good fit.  same with `qparam_broadcast_op`

Reply:
ah forgot to update the param name, Ill update it